README.md

```diff
@@ -119,6 +118,8 @@ const jmonData = new JMON(INPUT_DATA);
   ```javascript
   jmon.commit();
   ```
+### Errors and Exception:
+ - JMON initialization with any primitive types will throw an `Error`. <b>It only supports JSON and Arrays of JSON </b>
```

Comment:
Can you also add `object` after `Error`

JMON initialization with any primitive types will throw an `Error` object. <b>It only supports JSON and Arrays of JSON </b>

Reply:
sure